packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Router } from '@kbn/core-http-router-server-internal';
+import { getResponseValidation } from '@kbn/core-http-server';
+import { ALLOWED_PUBLIC_VERSION as LATEST_SERVERLESS_VERSION } from '@kbn/core-http-router-server-internal';
+import type { OpenAPIV3 } from 'openapi-types';
+import type { OasConverter } from './oas_converter';
+import {
+  assignToPathsObject,
+  extractContentType,
+  extractValidationSchemaFromRoute,
+  getPathParameters,
+  getVersionedContentTypeString,
+  getVersionedHeaderParam,
+  prepareRoutes,
+} from './util';
+import type { OperationIdCounter } from './operation_id_counter';
+
+export const processRouter = (
```

Comment:
I assume this was extracted from `packages/kbn-router-to-openapispec/src/generate_oas.ts` without much changes?

Reply:
Exactly. Contains this fix:

```
Fixed an issue where we might not be iterating over response codes correctly (and updated test cases)
```